### PR TITLE
Default Template for Testing Update Widgets

### PR DESCRIPTION
**Pull request by adolfdaniel**
Add a default template that increments a counter on Action.Execute.

### CODE REVIEW

File: widgets/sw.js [hunk 1]
```diff
@@ -1,3 +1,14 @@
+const defaultActionVerb = 'inc';
+const defaultTemplate = { "type": "AdaptiveCard", "body": [{ "type": "TextBlock", "text": "You have clicked the button ${count} times" }], "actions": [{ "type": "Action.Execute", "title": "Increment", "verb": `${defaultActionVerb}`, "style": "positive" }], "$schema": "http://adaptivecards.io/schemas/adaptive-card.json", "version": "1.5" };
```
Comment (amandabaker) [inline widgets/sw.js]: Can this be formatted into multiple lines?

File: widgets/sw.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+let count = 0;
```
Comment (amandabaker) [inline widgets/sw.js]: nit: Consider `clickCount`/`buttonClickCount` so it's clear what we're counting

**Review by amandabaker — COMMENTED** [review]
Just a couple readability nits. I can fix after if you don't have time to make them now

File: widgets/sw.js (hunk 1 above)
Comment (adolfdaniel) [reply widgets/sw.js]: Done.

File: widgets/sw.js (hunk 2 above)
Comment (adolfdaniel) [reply widgets/sw.js]: Done.

**Pull request merged**